nss-sapphire-elephants/BangazonAPI
# Customer

# Description
Added parameters to Customer model.

Added GET, PUT, POST methods to Customer controller.

Created  CustomerTest.cs with GET all, GET single, POST and PUT method testing.

Added parameters to Employee model.

Added GET PUT POST methods to Employee controller.

Created EmployeeTest.cs with GET all, GET single, POST, PUT method testing.

## Type of change
  New feature (non-breaking change which adds functionality)

# Testing Instructions for Change Made
go to master
git fetch --all
git checkout customer
Run the application
Ensure GET, POST, PUT function properly in Postman
for http://localhost:5000/api/employee
for http://localhost:5000/api/customer
Run the CustomerTest.cs tests and ensure all are successful
Run the EmployeeTest.cs tests and ensure all are successful

# Checklist:
